In-house patch to include options defined in nova.network.neutronv2.api
that are missing from the nova.conf generated by
tools/config/generate_sample.sh. Some of these options are deprecated,
although valid, in Kilo but they are removed in Liberty.

--- nova-2015.1.2/nova/network/neutronv2/api.py.orig	2016-03-22 22:03:30.626104765 -0700
+++ nova-2015.1.2/nova/network/neutronv2/api.py	2016-03-22 22:19:18.869889129 -0700
@@ -142,6 +142,7 @@ _ADMIN_AUTH = None
 
 def list_opts():
     list = copy.deepcopy(_neutron_options)
+    list.extend(neutron_opts)
     list.insert(0, auth.get_common_conf_options()[0])
     # NOTE(dims): There are a lot of auth plugins, we just generate
     # the config options for a few common ones
